README.md

```diff
@@ -1,20 +1,6 @@
-## devenv
+Purpose: enable `gh act` to work quickly on localhost
 
-the next generation sentry devenv management tool
+To that end:
 
-## install
```

Comment:
why remove all this? i'm gonna be telling people to follow these install instructions from notion onboarding guide